crew

```diff
@@ -334,28 +334,31 @@ def remove (pkgName)
     return
   end
 
-  Dir.chdir CREW_CONFIG_PATH do
-
-    #remove all files installed by the package
-    File.open("meta/#{pkgName}.filelist").each_line do |line|
-      begin
-        File.unlink line.chomp
-      rescue => exception #swallow exception
+  #if the package is marked as 'is_fake', the file/directorylists doesn't exist
```

Comment:
Can you try to reproduce that? It shouldn't be possible.
If a package gets upgraded, the filelist and directorylist gets removed via https://github.com/skycocker/chromebrew/blob/956e5e58a4084d34f033aade2dfb9f7e4df4cd92/crew#L351-L352
And if the package has turned into a fake package, crew will not create new filelist/directorylists.